optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -146,7 +146,21 @@ def _metric_with_eval_at(self, metric: str) -> str:
     def higher_is_better(self) -> bool:
 
         metric_name = self.lgbm_params.get("metric", "binary_logloss")
-        return metric_name in ("auc", "auc_mu", "ndcg", "map", "average_precision")
+        supported_higher_is_better = [
+            "auc",
+            "auc_mu",
+            "ndcg",
+            "lambdarank",
+            "rank_xendcg",
+            "xendcg",
+            "xe_ndcg",
+            "xe_ndcg_mart",
+            "xendcg_mart",
+            "map",
+            "mean_average_precision",
+            "average_precision",
+        ]
+        return metric_name in supported_higher_is_better
```

Comment:
Could you revert this change, please? The metrics that you added (i.e., `landmark`, `rank_xendcg`, `xendcg`, `xe_ndcg`, `xe_ndcg_mart`, `xendcg_mart`) are converted to `ndcg` by `_handling_alias_parameters`.

https://github.com/optuna/optuna/blob/6568ef921b58b94e0eb2d427e60c4c28d4c74ac0/optuna/integration/_lightgbm_tuner/alias.py#L30-L55

Reply:
Right, I didn't notice the handling of aliases. I fixed it in the last commit.

Reply:
Thanks!